src/versionInfo.ts

```diff
@@ -0,0 +1,24 @@
+export default {
```

Comment:
Does the script that generates this file know that it should generate a `.ts` file in the future?

Reply:
Yes, you can see the script `pre` in package.json

Reply:
Ah, oops, missed that - thanks!